src/main/java/org/broadinstitute/hellbender/tools/spark/sv/RunMinimalBWAMEM.java

```diff
@@ -0,0 +1,193 @@
+package org.broadinstitute.hellbender.tools.spark.sv;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.io.IOUtils;
+import org.broadinstitute.hellbender.cmdline.Argument;
+import org.broadinstitute.hellbender.cmdline.CommandLineProgram;
+import org.broadinstitute.hellbender.cmdline.CommandLineProgramProperties;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.cmdline.programgroups.StructuralVariationSparkProgramGroup;
+import org.broadinstitute.hellbender.exceptions.GATKException;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import scala.Tuple3;
+
+import java.io.*;
+import java.nio.charset.Charset;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.ArrayList;
+
+// TODO: choose which parameters allowed to be tunable
+// TODO: if throws, would temp files be cleaned up automatically?
+@CommandLineProgramProperties(
+        summary        = "Minimal program to call BWAMEM for performing alignment, allowing limited options.",
+        oneLineSummary = "Minimal work to call BWAMEM.",
+        programGroup   = StructuralVariationSparkProgramGroup.class)
+public final class RunMinimalBWAMEM extends CommandLineProgram {
+
+    @Argument(doc       = "Absolute path to BWA program.",
+              shortName = "bwaPath",
+              fullName  = "fullPathToBWA",
+              optional  = false)
+    public String pathToBWA = null;
+
+    @Argument(doc       = "Absolute path to input FASTQ/A file to be aligned.",
+              shortName = StandardArgumentDefinitions.INPUT_SHORT_NAME,
```

Comment:
dont use -I and -I2 for this - use positional arguments. See `CompareBaseQualities` for an example